### ISSUE TITLE
Unable to plot points in sketch area with Google Chrome v116.0.5845.97

### ISSUE THREAD

**Issue by Uminosachi** [gradio-app/gradio#5241]
### Describe the bug

When using Google Chrome version 116.0.5845.97 with the gradio.Image component set tool="sketch", users are unable to plot points in the sketch area. This issue is specific to Chrome as the feature works correctly in both Microsoft Edge and Firefox browsers.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr
block = gr.Blocks().queue()
with block:
    sketch_image = gr.Image(label="sketch image", tool="sketch", interactive=True)

block.launch()
```

* Steps to Reproduce
  - In the generated interface, upload an image to the "sketch image" component.
  - Click within the sketch area.

* Expected Result
  - A point should be plotted in the sketch area upon clicking.

* Actual Result
  - Unable to plot points.

### System Info

```shell
Gradio version: 3.40.1
Browser: Google Chrome version 116.0.5845.97
OS: Windows and Linux
```

### Severity

I can work around it

**Comment by YuyanHuang** [comment]
I have the same problem.
Browser: Google Chrome version 116.0.5845.96

**Comment by Uminosachi** [comment]
The same issue is observed in Microsoft Edge version 116.0.1938.62.

**Comment by Uminosachi** [comment]
This issue has been fixed in Google Chrome version 116.0.5845.187, so I will close this issue.

**Comment by ControllableGeneration** [comment]
Still have this problem with version 124.0.6367.91
